ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -43,3 +46,40 @@ tests:
   commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-frontend.sh
   container:
     from: test-bin
+
+#  builder_run 'GUI-Tests' 1 ./test-gui.sh crud
```

Comment:
I'd remove all the comments about old jobs since they'll just go away. Later on the comments will be more confusing than helpful.

Reply:
Yup, going to do that once I'm ready to remove the WIP.